### PR TITLE
Update cache.md

### PR DESCRIPTION
**Pull request by ruudboon**
Xcache is not in Pecl anymore

### CODE REVIEW

**Comment by CameronHall** [comment]
Thanks @ruudboon. Can you please update the link to somewhere people may get the extension? Perhaps this? https://xcache.lighttpd.net/#Downloads

File: en/cache.md [hunk 1]
```diff
@@ -458,7 +458,7 @@ The backend adapters available to store cache data are:
 | [Phalcon\Cache\Backend\Memory](api/Phalcon_Cache_Backend_Memory)       | Stores data in memory                                |                                           |                                                    |
 | [Phalcon\Cache\Backend\Mongo](api/Phalcon_Cache_Backend_Mongo)        | Stores data to Mongo Database.                       | [MongoDB](https://mongodb.org/)            | [Mongo](https://mongodb.org/)                       |
 | [Phalcon\Cache\Backend\Redis](api/Phalcon_Cache_Backend_Redis)        | Stores data in Redis.                                | [Redis](https://redis.io/)                 | [Redis](https://pecl.php.net/package/redis)         |
-| [Phalcon\Cache\Backend\Xcache](api/Phalcon_Cache_Backend_Xcache)       | Stores data in XCache.                               | [XCache](https://xcache.lighttpd.net/)     | [XCache](https://pecl.php.net/package/xcache)       |
+| [Phalcon\Cache\Backend\Xcache](api/Phalcon_Cache_Backend_Xcache)       | Stores data in XCache.                               | [XCache](https://xcache.lighttpd.net/)     | XCache                                              |
```
Comment (CameronHall) [inline en/cache.md:461]: Can you please update the link to somewhere people may get the extension? Perhaps this? https://xcache.lighttpd.net/#Downloads

**Review by CameronHall — CHANGES_REQUESTED** [review]

**Comment by niden** [comment]
I have not rewritten the cache component (still WIP). Once I do this document will be updated to the latest functionality. I belive Apc and Xcache have been deprecated for this component

**Comment by CameronHall** [comment]
In that case, we can make changes to this page later on when @niden is done with the new cache component.

**Pull request closed without merging**